Resources/config/checks/apc_fragmentation.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+return static function (ContainerConfigurator $container) {
+    $services = $container->services();
+    $parameters = $container->parameters();
+
+    $services->set('liip_monitor.check.apc_fragmentation', \Laminas\Diagnostics\Check\ApcFragmentation::class)
+        ->public()
+        ->args([
+            '%%liip_monitor.check.apc_fragmentation.warning%%',
+            '%%liip_monitor.check.apc_fragmentation.critical%%',
```

Comment:
I'm surprised by the double `%%`? They look like parameters, and `%%` is for escaped `%`.

Numeric values are [expected](https://github.com/laminas/laminas-diagnostics/blob/1.30.x/src/Check/ApcFragmentation.php#L51-L63).
```suggestion
            param('liip_monitor.check.apc_fragmentation.warning'),
            param('liip_monitor.check.apc_fragmentation.critical'),
```